### ISSUE TITLE
New step: `put_aws_sigv4`

### ISSUE THREAD

**Issue by wojtekmach** [wojtekmach/req#62]
Similar to https://curl.se/docs/manpage.html#--aws-sigv4.

```elixir
iex> Req.put!(url, body: body, aws_sigv4: [region: "us-east-1", service: "s3"])
```

**Comment by wojtekmach** [comment]
Closing in favour of PR.

**Comment by wojtekmach** [comment]
:eyes https://github.com/lpil/aws4_request

**Comment by wojtekmach** [comment]
Released in v0.4.9. Thanks @aleDsz again, it took a while but we got there! :)
